src/pkg/cli/client/byoc/aws/byoc.go

```diff
@@ -326,8 +326,11 @@ func (b *ByocAws) environment() map[string]string {
 	}
 }
 
-func (b *ByocAws) runCdCommand(ctx context.Context, cmd ...string) (ecs.TaskArn, error) {
+func (b *ByocAws) runCdCommand(ctx context.Context, extraEnv map[string]string, cmd ...string) (ecs.TaskArn, error) {
```

Comment:
there's no reason to deal with `map` here since it's only used by us, so let's just use a typed `behavior` parameter